Framework/API/test/AsynchronousTest.h

```diff
@@ -11,12 +11,21 @@
 #include <Poco/NObserver.h>
 #include <Poco/Thread.h>
 
+#include <condition_variable>
+#include <mutex>
+
 using namespace Mantid::Kernel;
 using namespace Mantid::API;
 using namespace std;
```

Comment:
Why do we need to use `DECLARE_ALGORITHM` at all if we are instantiating the object directly?